devicon.json

```diff
@@ -4248,6 +4248,31 @@
         "color": "#1FA6CA",
         "aliases": []
     },
+    {
+        "name": "materializecss",
+        "altnames": [
+            "materialize",
+            "materialize-css",
+            "materialize css"
+        ],
+        "tags": [
+            "framework",
+            "css",
+            "design",
+            "material-design",
+            "framework"
```

Comment:
Seems this tag is repeated. 👀
```suggestion
            "material-design"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Oh I see.